app/views/admin/appointments/index.html.erb

```diff
@@ -5,16 +5,16 @@
 <div class="columns">
   <div class="col-12" style="justify-content: center;">
     <ul class="pagination">
-      <li class="page-item page-prev">
```

Comment:
I see we've already got some inline `style` tags in this template, but can you please move the new styles into a `scss` file?

Reply:
Sure, I will do that

Reply:
It is done. @jim